bigchaindb/consensus.py

```diff
@@ -162,6 +171,14 @@ def validate_transaction(bigchain, transaction):
                     raise exceptions.DoubleSpend(
                         'input `{}` was already spent'.format(fulfillment['input']))
 
+                txids.append(fulfillment['input']['txid'])
```

Comment:
This is related to https://github.com/bigchaindb/bigchaindb/pull/599#discussion-diff-78031204

`get_asset_id` is used in `create_tx` and `validate_transaction`. In `validate_transaction` its true that I already have the input transactions but in `create_tx` I don't. I will need to retrieve the input transactions in `create_tx` before calling `get_asset_id`